llvm/test/Analysis/ScalarEvolution/predicated-symbolic-max-backedge-taken-count.ll

```diff
@@ -0,0 +1,119 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes='print<scalar-evolution>' %s -disable-output 2>&1 | FileCheck %s
```

Comment:
Use `-scalar-evolution-classify-expressions=0` to only print the exit counts.

Reply:
Ah thanks, added in the commit that landed the tests ( ed4227aad37f2c4adf307b63050fb9aee52b07f8 )